### ISSUE TITLE
Combo of array indeces and suffixes not parsing right.

### ISSUE THREAD

**Issue by Dunbaratu** [KSP-KOS/KOS#224]
In the act of trying to test out a KSPField reading feature, I came across this other problem and wanted to log it:

When this works:
```
   set myvar to mylist[3]:somesuffixname.
   print myvar[0].
```
then this should, logically, also work and be the same thing:
```
   print mylist[3]:somesuffixname[0].
```
But the parser seems to choke on that mixture of array index and suffixes in one long chain.  It claims the second way is a syntax error.

**Comment by abenkovskii** [comment]
It looks like it is fixed. Isn't it? This code works fine:
```
print ship:parts[0]:modules[3].
```

**Comment by hvacengi** [comment]
Yes, this was fixed in #938
